docs/API/Users_Guide/Data_Analysis.md

```diff
@@ -9,6 +9,9 @@ The following data analysis endpoints are available from the GDC API:
 |__Node__| __Endpoint__ | __Description__ |
 |---|---|---|
 |__Genes__| __/genes__ | Allows users to access summary information about each gene using its Ensembl ID. |
+|__Gene Expression__|__/gene_expression/availability__|Allows users to retrieve the availability of gene expression data for specific cases and/or genes.|
+||__/gene_expression/values__|Get gene expression values for specified cases and genes.|
+||__/gene_expression/gene_selection__|Allows users to select the most variably expressed genes for a collection of cases and genes.|
```

Comment:
```suggestion
||__/gene_expression/gene_selection__|Select the most variably expressed genes for a collection of cases and genes.|
```
I recommend this change because the endpoint performed automatic selection so the user does not have to.

Reply:
changed, thank you for the comment!